src/plots.jl

```diff
@@ -281,15 +283,19 @@ mutable struct Image <: Plot
         else
             write(ColorMaps.RGBArrayMap(colormap), A, filename)
         end
-        new(filename, xrange[1], xrange[2], yrange[1], yrange[2], zmin, zmax, colorbar, colormap, style)
+        if colorbarStyle == "{ymode=log, scaled ticks = false}"
```

Comment:
I don't think we want this check, otherwise it breaks our ability to customize the style. I think we want to branch on `zmode == "log"`. But, `zmode` is not part of Image at the moment, so you'll want to add this field and make sure it is passed properly from `Histogram2`.

Reply:
ill do that now!